### DISCUSSION TITLE
How to increase the height of the PasswordInput component?

### DISCUSSION THREAD

**Discussion by dmitry-ff** [mantinedev/mantine#5240] [Q&A]
If you use the prop `size`, then the font size increases along with the height. There is a way in the theme object of the `components` property of the Inputa component to set the height and font size through the `--input-height` and `--input-fz` variables, but this option does not suit me.
Is it possible to use Input.extend outside of a theme object, if so how?

**Comment by rtivital** [comment]
https://codesandbox.io/s/patient-fire-2f9vrf?file=/src/App.tsx
